### ISSUE TITLE
[Feature Request] Elementor Editor: Section and Column Tabs - configuration

### ISSUE THREAD

**Issue by nickdgreen** [pojome/elementor#1615]
I would like to be able to set in the Elementor Settings (within the editor Hamburger menu) The size and color of the section and column tab's text and background - this can be achieved with some CSS provided by Lyle Chamney but it would be great to do it without css for the css averse. Additionally it would greate to be able to select the position of the tabs top, left, right, bottom as sometimes the styling that has been applied to an element (on purpose or in error) means the tabs cannot be accessed.

**Comment by arielk** [comment]
I can not figure out exactly what you mean, I'd be happy if you could explain better

**Comment by nickdgreen** [comment]
Sorry having read my description it is pretty confusing (Doh!). Let me try again.  Sometimes, depending on the background of the page area you are editing it is difficult to the Section and Column tabs of the section you are editing. This is made worse if the section has a negative margin applied to it as the tabs become inaccessible. A "fix" has been supplied by a member that allows you to use some CSS, in for example the simple CSS plugin, that changes the position, size and colours of the Section and Column Tabs. For those that are comfortable with CSS this is OK but to turn the restyling on and of the CSS has to be either added then deleted or added and "rem'd" out when not required. 

My request is that functionality be added to the Elementor Settings accessed through the 'hamburger' menu in the Editor (Global Colours, Global Fonts, Color Picker etc.) that would enable the configuration of the Section and Column Tabs: A default check box that unchecked would enable configuration of - Tab Background Colour, Tab Text Colour, Font Size, Tab Position (Top, Right, Bottom, Left). In this way, non-CSS users could access tabs that have become "hidden" or make tabs more visible to them if they have vision impairments. The default checkbox would make it easy to turn on and off.
![elementor setting](https://cloud.githubusercontent.com/assets/22448787/25566094/05444cc4-2dcb-11e7-9334-66123972666e.png)

I hope this makes more sense.
